test/runner/reporter.js

```diff
@@ -57,30 +68,35 @@ export function reportTest( test, reportId, { browser, headless } ) {
 						diff = [ { removed: true, value: `${ value }` } ];
 					}
 				} else if (
-					typeof error.expected === "boolean" &&
-					typeof error.actual === "boolean"
+					typeof error.expected === "string" &&
```

Comment:
BTW, there's a typo in the number part - it checks for a `number` typeof but it checks `error.expected` twice.